turbopuffer/query.py

```diff
@@ -1,18 +1,17 @@
 from dataclasses import dataclass
 import sys
 from typing import Optional, List, Tuple, Union, Dict
-from enum import Enum
 
+# Refer to turbopuffer docs for valid operator names
+FilterOperator = str
```

Comment:
Yeah I guess this is a bit annoying to type out, and not too valuable? We do this in the TS client though iirc, but I am totally fine with this

Reply:
we were missing over half of our operators because we didn't update this when adding them, I think better to avoid the drift